pkg/cmd/server/logs.go

```diff
@@ -30,7 +31,10 @@ var logsCmd = &cobra.Command{
 
 		query := ""
 		if followFlag {
-			query = "follow=true"
+			query += "follow=true"
+		}
+		if retryFlag {
+			query += "&retry=true"
```

Comment:
This is not correct for the case where follow is not supplied. In that case you would have `query = "&retry=true`